packages/pagebuilder/lib/ContentTypes/Products/products.css

```diff
@@ -26,7 +26,7 @@
 }
 
 .galleryItems {
-    composes: items from '../../../../Gallery/gallery.css';
+    composes: items from '~@magento/venia-ui/lib/components/Gallery/gallery.css';
```

Comment:
Accidental tilde?

Reply:
> Accidental tilde?

Not accidental. Required for absolute `node_modules` refs since [`css-loader@2.0.0`](https://github.com/webpack-contrib/css-loader/tree/v2.0.0#url).